### ISSUE TITLE
Nested unprefixed elements not properly serialized

### ISSUE THREAD

**Issue by nikku** [bpmn-io/bpmn-moddle#38]
We do not serialize the following stuff right now which is valid XML:

``` xml
<bpmn:extensionElements>
  <foo:config>
    <Entry key="A" value="B" />
    <Nested>
      <Entry key="A" value="B" />
    </Nested>
  </foo:config>
</bpmn:extensionElements>
```

When attempting, the export just blows up.

**Comment by nikku** [comment]
Related to camunda/camunda-modeler#130

**Comment by nikku** [comment]
This prevents us from updating the [miwg-test-suite](https://github.com/bpmn-io/bpmn-miwg-test-suite) to the latest version, too.

**Comment by nikku** [comment]
This gets addressed via https://github.com/bpmn-io/moddle-xml/issues/23.

**Comment by nikku** [comment]
With bpmn-io/moddle-xml#24 implemented we're able to pass these unprefixed elements through without throwing an error.

**Comment by nikku** [comment]
:cake:

**Comment by nikku** [comment]
Fixed with `moddle-xml@1.1.1`.
